tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -110,24 +110,37 @@ spec:
         echo "${pyxisKey}" > /tmp/key
 
         COMPONENTS=$(jq '.components | length' "${SNAPSHOT_SPEC_FILE}")
+        JSON_OUTPUT='{}'
         for (( i=0; i < $COMPONENTS; i++ )); do
             CONTAINER_IMAGE=$(jq -r ".components[${i}].containerImage" "${SNAPSHOT_SPEC_FILE}")
+            JSON_OUTPUT=$(jq ".components[$i] += { \"containerImage\": \"${CONTAINER_IMAGE}\", \"pyxisImages\": []}" \
+                              <<< "$JSON_OUTPUT")
             REPOSITORY=$(jq -r ".components[${i}].repository" "${SNAPSHOT_SPEC_FILE}")
             REPOSITORY=${REPOSITORY%:*} # strip tag just in case - it should not be there
             DIGEST="${CONTAINER_IMAGE##*@}"
             PULLSPEC="docker://${REPOSITORY}@${DIGEST}"
             MEDIA_TYPE=$(skopeo inspect --raw "$PULLSPEC" | jq -r .mediaType)
-            skopeo inspect --no-tags "$PULLSPEC" > /tmp/skopeo-inspect.json
+            ARCHS=($(get-image-architectures "${PULLSPEC#*://}"))
```

Comment:
I wonder if we could streamline the PULLSPEC variable here. We first define it by adding "docker://" as a prefix. And now we use it twice with "docker://" and once without - by removing it again.

Maybe it would be nicer to have the variable without "docker://" and add it on the two occasions where it's needed, rather then removing it in this case? WDYT?

So `PULLSPEC="${REPOSITORY}@${DIGEST}"` and then just add the prefix where needed.

Reply:
I agree, so I removed the "docker://"  from the PULLSPEC declaration.

Reply:
Cool, thanks.